core/cap-0035.md

```diff
@@ -378,6 +402,19 @@ If an `AllowTrustOp` operation is submitted with the `CLAWBACK_ENABLED_FLAG`
 set, the operation will fail with the existing result code
 `ALLOW_TRUST_MALFORMED`.
 
+#### Set Options Operation
+
+This proposal introduces no changes to the `SetOptionsOp` operation XDR, but
+the operation will require the `AUTH_REVOCABLE_FLAG` flag to be set whenever
+the `AUTH_CLAWBACK_ENABLED_FLAG` flag is set.
+
+This introduces a failing result in these cases:
+
+1. Setting only `AUTH_CLAWBACK_ENABLED_FLAG` without `AUTH_REVOCABLE_FLAG`
+   already set will result in `SET_OPTIONS_BAD_FLAGS`.
+2. Clearing `AUTH_REVOCABLE_FLAG` while AUTH_CLAWBACK_ENABLED_FLAG is set
+   will result in SET_OPTIONS_BAD_FLAGS.

```

Comment:
Missing code block for SET_OPTIONS_BAD_FLAGS.

Reply:
It's an existing result code. Not new.

Reply:
I misunderstood you, sorry. This is fixed in 1f8c049.

Reply:
I meant there are no quotes around the SET_OPTIONS_BAD_FLAGS in part 2, while the same result has them in part 1.